niviz_rater/validation.py

```diff
@@ -1,47 +1,78 @@
+import os
 import yaml
 import json
-import os
 import yamale
 from yamale.validators import DefaultValidators, Validator
-from bids.config import get_option
 
 SCHEMAFILE = os.path.join(os.path.dirname(__file__), 'data/schema.yaml')
 
 
-def load_json(file):
```

Comment:
[nit] Having this be accessible by other modules doesn't seem like a bad thing - loading jsons is pretty common. I think this could go into a utils file if you had one, but making it just for this seems like overkill

Reply:
Let's do that!